### ISSUE TITLE
postinstall in `bun add` does not behave the same as `bun install`

### ISSUE THREAD

**Issue by lowkahonn** [oven-sh/bun#4891]
### What version of Bun is running?

1.0.0+822a00c4d508b54f650933a73ca5f4a3af9a7983

### What platform is your computer?

Darwin 22.3.0 arm64 arm

### What steps can reproduce the bug?

I have created a minimal reproducible project here: https://github.com/lowkahonn/bun-add-bug/tree/main.
1. Go to https://github.com/lowkahonn/bun-add-bug/tree/main
2. Run `bun install` at the project root
3. Go to one of the workspace `cd packages/package-a`
4. Add a dependency `bun add react`

### What is the expected behavior?

`bun add` should be able to find the script when running postinstall.

```sh
bun install v1.0.0 (822a00c4)
 + package-a@workspace:packages/package-a
Empty postinstall script

 1 packages installed [59.00ms]
  Removed: 1
```

### What do you see instead?

```sh
bun add v1.0.0 (822a00c4)

 installed react@18.2.0

bash: ./script.sh: No such file or directory

 3 packages installed [141.00ms]
  Removed: 1
```

### Additional information

The above error causes `bun add` to fail and thus the packages are not installed despite it shows that they are installed

**Comment by emilycares** [comment]
In bun postinstall is not executed by default. You have to enable it.
https://bun.sh/docs/cli/install#trusted-dependencies

**Comment by lowkahonn** [comment]
@micmine It seems like postinstall is executed for packages in the workspaces. I've also tried added the package to `trustedDependencies`, but running `bun add` in the workspace still fails because current working directory is at the root

**Comment by ghiscoding** [comment]
> @micmine It seems like postinstall is executed for packages in the workspaces. I've also tried added the package to `trustedDependencies`

I just saw that a core team member created a similar issue #4959 about trusted dependencies

> but running `bun add` in the workspace still fails because current working directory is at the root

also found a similar but newer issue #4935 with a link to a much older issue #533 which is the main monorepo tracking issue

_Note: I did not even try Bun yet because I'm on Windows, but I like to look at issues and PRs every now and then_

**Comment by dylan-conway** [comment]
Fixed in #11177 and available in `v1.1.10`. If you continue to have issues with workspaces please reopen this issue or open a new one.
